media-gfx/graphviz/patches/graphviz-2.40.1.patchset

```diff
@@ -0,0 +1,120 @@
+From 0da7cfb6d8cb0c66675f769c19b4464d6eb8e052 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sun, 25 Feb 2018 15:09:25 +0100
+Subject: set Haiku font directories
+
+
+diff --git a/configure.ac b/configure.ac
+index 51166c3..59c89a5 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -133,6 +133,7 @@ UWIN=no
+ CYGWIN=no
+ MINGW32=no
+ DARWIN9=no
++HAIKU=no
+ case "${host_os}" in
+   *uwin* )
+ 	UWIN=yes
+@@ -181,6 +182,16 @@ case "${host_os}" in
+ 		;;
+         esac
+ 	;;
++	*haiku* )
++	HAIKU=yes
++	HAIKU_SYSTEM_FONTPATH=`finddir B_SYSTEM_FONTS_DIRECTORY`
++	HAIKU_COMMON_FONTPATH=`finddir B_COMMON_FONTS_DIRECTORY`
++	HAIKU_USER_FONTPATH=`finddir B_USER_FONTS_DIRECTORY`
++	DEFAULT_FONTPATH="$HAIKU_SYSTEM_FONTPATH:$HAIKU_SYSTEM_FONTPATH/ttfonts:$HAIKU_SYSTEM_FONTPATH/psfonts:$HAIKU_COMMON_FONTPATH:$HAIKU_COMMON_FONTPATH/ttfonts:$HAIKU_COMMON_FONTPATH/psfonts:$HAIKU_USER_FONTPATH:$HAIKU_USER_FONTPATH/ttfonts:$HAIKU_USER_FONTPATH/psfonts"
++	PATHSEPARATOR=":"
++        NO_POSTSCRIPT_ALIAS=1
++	BROWSER="open"
++	;;
+   * ) # This sucks!  Whatever happened to standards for file system layout?   (Good job its no longer used unless fontconfig is unavailable.)
+ 	DEFAULT_FONTPATH="/usr/share/fonts/sun/TrueType:/usr/share/fonts/TrueType:/usr/X11/lib/X11/fonts/TrueType:/usr/share/fonts/sun/Type1:/usr/openwin/lib/X11/fonts/Type1:/usr/share/fonts/X11/Type1:/usr/share/fonts/default/Type1:/usr/X11R6/lib/X11/fonts/TrueType:/usr/X11R6/lib/X11/fonts/truetype:/usr/X11R6/lib/X11/fonts/TTF:/usr/share/fonts/truetype:/usr/openwin/lib/X11/fonts/TrueType:/usr/X11R6/lib/X11/fonts/Type1:/usr/common/share/fonts/ttf";
+ 	PATHSEPARATOR=":"
+@@ -422,7 +433,7 @@ AC_CHECK_HEADERS(malloc.h \
+ 	errno.h unistd.h strings.h \
+ 	sys/time.h sys/types.h sys/select.h \
+ 	sys/stat.h sys/mman.h \
+-	sys/ioctl.h sys/inotify.h crt_externs.h)
++	sys/ioctl.h sys/inotify.h crt_externs.h sysexits.h)
+ AC_HEADER_TIME
+ AC_HEADER_DIRENT
+ AC_HEADER_STDBOOL
+-- 
+2.16.2
+
+
+From 2aef2817615366f4b93da74a1bbd7001c1a525c0 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sun, 25 Feb 2018 15:12:56 +0100
+Subject: Don't define waitpid unless necessary
+
+
+diff --git a/lib/sfio/sfhdr.h b/lib/sfio/sfhdr.h
+index 9558c03..0fa8d73 100644
+--- a/lib/sfio/sfhdr.h
++++ b/lib/sfio/sfhdr.h
+@@ -906,7 +906,9 @@ extern "C" {
+ #endif
+ 
+     extern time_t time(time_t *);
+-    extern int waitpid(int, int *, int);
++#ifndef waitpid
++    extern pid_t waitpid(pid_t, int *, int);
++#endif
```

Comment:
Did you merge both patches together? Only one of them should be necessary.

Reply:
So the #ifndef waitpid/#endif shouldn't be needed?